### PR TITLE
Record status for various features

### PR DESCRIPTION
**Pull request by jcouv**
FYI @cston

### CODE REVIEW

**Review by RikkiGibson — APPROVED** [review]

**Review by cston — APPROVED** [review]

**Comment by jcouv** [comment]
FYI made some other updates to track three other features on the radar

**Review by cston — APPROVED** [review]

**Review by RikkiGibson — APPROVED** [review]

File: docs/Language Feature Status.md [hunk 1]
```diff
@@ -24,9 +24,12 @@ efforts behind them.
...
-| [Switch on ReadOnlySpan<char>](https://github.com/dotnet/csharplang/issues/1881) | main | [In Progress](https://github.com/dotnet/roslyn/pull/44388) | [YairHalberstadt ](https://github.com/YairHalberstadt) | [cston](https://github.com/cston), [RikkiGibson](https://github.com/RikkiGibson) | [jcouv](https://github.com/jcouv) |
+| [Switch on ReadOnlySpan<char>](https://github.com/dotnet/csharplang/issues/1881) | main | [In Progress](https://github.com/dotnet/roslyn/issues/59191) | [YairHalberstadt ](https://github.com/YairHalberstadt) | [cston](https://github.com/cston), [RikkiGibson](https://github.com/RikkiGibson) | [jcouv](https://github.com/jcouv) |
 | [nameof accessing instance memebers](https://github.com/dotnet/roslyn/issues/40229) | main | [In Progress](https://github.com/dotnet/roslyn/pull/48754) | [YairHalberstadt ](https://github.com/YairHalberstadt) | [333fred](https://github.com/333fred), [AlekseyTs](https://github.com/AlekseyTs)  | [333fred](https://github.com/333fred) |
```
Comment (cston) [inline docs/Language Feature Status.md:28]:
```suggestion
| [nameof accessing instance members](https://github.com/dotnet/roslyn/issues/40229) | main | [In Progress](https://github.com/dotnet/roslyn/pull/48754) | [YairHalberstadt ](https://github.com/YairHalberstadt) | [333fred](https://github.com/333fred), [AlekseyTs](https://github.com/AlekseyTs)  | [333fred](https://github.com/333fred) |
```

File: docs/Language Feature Status.md [hunk 2]
```diff
@@ -24,9 +24,12 @@ efforts behind them.
...
+| [nameof accessing instance members](https://github.com/dotnet/roslyn/issues/40229) | main | [In Progress](https://github.com/dotnet/roslyn/pull/48754) | [YairHalberstadt ](https://github.com/YairHalberstadt) | [333fred](https://github.com/333fred), [AlekseyTs](https://github.com/AlekseyTs)  | [333fred](https://github.com/333fred) |
 | [Utf8 String Literals](https://github.com/dotnet/csharplang/issues/184) | [Utf8StringLiterals](https://github.com/dotnet/roslyn/tree/features/Utf8StringLiterals) | [In Progress](https://github.com/dotnet/roslyn/issues/58848) | [AlekseyTs](https://github.com/AlekseyTs) | [cston](https://github.com/cston), [RikkiGibson](https://github.com/RikkiGibson) | [MadsTorgersen](https://github.com/MadsTorgersen) |
+| [Attributes on Main for top level programs](https://github.com/dotnet/csharplang/issues/5045) | TBD | [In Progress](https://github.com/dotnet/roslyn/issues/59193) | TBD | TBD | [jaredpar](https://github.com/jaredpar) |
```
Comment (jaredpar) [inline docs/Language Feature Status.md:30]:
```suggestion
| [Attributes on Main for top level programs](https://github.com/dotnet/csharplang/issues/5045) | [chsienki](https://github.com/chsienki) | [In Progress](https://github.com/dotnet/roslyn/issues/59193) | TBD | TBD | [jaredpar](https://github.com/jaredpar) |
```

**Review by jaredpar — APPROVED** [review]

File: docs/Language Feature Status.md (hunk 2 above)
Comment (jcouv) [reply docs/Language Feature Status.md]: Oops. Removed that line. It was dupe from "Top Level statement attribute specifiers" above.

**Pull request merged**